### PR TITLE
Create book page

### PR DESCRIPTION
**Pull request by martinmona**
## Summary
- Completed book page
- Added details

## Screenshot
image

## Trello Card
https://trello.com/c/wnYgIeRJ/26-detalle-de-libro-i-textos

### CODE REVIEW

File: Maquetado/index.html [hunk 1]
```diff
@@ -1,13 +1,31 @@
 <!doctype html>
 <html lang="en">
+
```
Comment (manueltuero) [inline Maquetado/index.html]: Remove empty line

File: Maquetado/index.html [hunk 2]
```diff
@@ -1,13 +1,31 @@
...
 <head>
   <meta charset="utf-8">
-  <title>HTML & CSS Training</title>
+  <title>Book details</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
```
Comment (manueltuero) [inline Maquetado/index.html]: Same

File: Maquetado/index.html [hunk 3]
```diff
@@ -1,13 +1,31 @@
...
+      <div class="book-item">
+        <span class="item-key">Editorial:</span> <span class="item-value">Nombre de la editorial</span>
+      </div>
+      <div class="book-item">
+        <span class="item-key">Año de publicación:</span> <span class="item-value">Año de publicación</span>
+      </div>
+    </div>
+  </div>
 </body>
+
```
Comment (manueltuero) [inline Maquetado/index.html]: Same

File: Maquetado/scss/application.scss [hunk 4]
```diff
@@ -1,2 +1,8 @@
 @import 'equalizer';
-@import 'styles';
+@import 'colors';
+@import 'details';
+
+*{
```
Comment (manueltuero) [inline Maquetado/scss/application.scss]: Add a space between `*` and `{`

File: Maquetado/scss/details.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
+.book {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-transparent;
+  display: flex;
+  flex-flow: column wrap;
+  height: 416px;
+  justify-content: flex-start;
+  margin: 0 auto;
+  max-width: 1000px;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Add `width: 100%;` rule

File: Maquetado/scss/details.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
+.book {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-transparent;
+  display: flex;
+  flex-flow: column wrap;
+  height: 416px;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Add `max-height: 415px;` and `height: 100%;`

File: Maquetado/scss/details.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  justify-content: flex-start;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: I think that this prop is unnecessary

File: Maquetado/scss/details.scss [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+.book-info {
+  margin: 10px auto;
+}
+
+.book-header {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: same

File: Maquetado/scss/details.scss [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.book-info {
+  margin: 10px auto;
+}
+
+.book-header {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: `flex-wrap: wrap;`

File: Maquetado/scss/details.scss [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+  justify-content: flex-start;
+  margin: 0 0 10px 0;
+}
+
+.book-item {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: same

File: Maquetado/scss/details.scss [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+  justify-content: flex-start;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: same

File: Maquetado/scss/details.scss [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+  margin: 50px auto;
+}
+
+.book-title {
+  color: $black;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+  margin: 0 5px 0 0;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Use `margin-right: 5px;` instead

File: Maquetado/scss/details.scss [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+  font-weight: 700;
+  line-height: 28px;
+}
+
+.item-key {
+  color: $black;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 24px;
+  margin: 0 5px 0 0;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: same

File: Maquetado/scss/details.scss [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+.book-title {
+  color: $black;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 38px;
+  margin: 0 5px 0 0;
+}
+
+.book-gender {
+  color: $grey;
```
Comment (manueltuero) [inline Maquetado/scss/details.scss]: Maybe you can avoid repeating CSS rules using a selector like `.book-gender, .item-value`

File: Maquetado/scss/details.scss [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.book-header {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 10px;
+}
+
+.book-item {
```
Comment (henryzarza) [inline Maquetado/scss/details.scss]: This properties are the same that `book-header` the only difference is `margin`. What do you think about of use `@extend` or selectors?

**Pull request merged**